service/controller/aws/v19/key/key.go

```diff
@@ -8,6 +8,12 @@ import (
 	"github.com/giantswarm/cluster-operator/pkg/v19/key"
 )
 
+// AppSpecs returns apps installed only for AWS.
+func AppSpecs() []key.AppSpec {
+	// Add any provider specific charts here.
+	return []key.AppSpec{}
+}
```

Comment:
On AWS and Azure we have provider specific apps so we need key functions for later.

Reply:
I thought we do the whole `app-operator` jazz in order to get rid of this kind of weirdness in the code. I am not following why we still have anything in `cluster-operator` defining which apps have to be run. I thought the catalogue is deployed somehow and there is just one generic app resource that does the magic for all apps. Or is this here only for migration?

Reply:
> I thought the catalogue is deployed somehow and there is just one generic app resource that does the magic for all apps. 

This is a nice idea, perhaps that we can think about it with @puja108.

Reply:
This is already in the spec :) https://github.com/giantswarm/giantswarm/blob/master/specs/app-catalog-phase-2.md#adding-managed-apps-to-release-crs

The apps will be defined in the release CRs and that covers having provider specific apps. The problem is we don't have the tooling yet for managing the release CRs.